api/src/services/user.service.ts

```diff
@@ -407,8 +407,7 @@ export class UserService {
       storedUser.userRoles?.isPartner;
     const isUserSiteMatch =
       (isPartnerPortalUser && dto.appUrl === process.env.PARTNERS_PORTAL_URL) ||
-      (!isPartnerPortalUser &&
-        dto.appUrl === storedUser.jurisdictions?.[0]?.publicUrl);
+      (!isPartnerPortalUser && dto.appUrl !== process.env.PARTNERS_PORTAL_URL);
```

Comment:
Still pondering if there are potential security issues here, where the appUrl doesn't need to be the/a publicUrl, it can be anything, as long as it's not the partners URL. I'm not sure I'm following the downside of a tighter match, if we were to .find on all of a user's jurisdictions

Reply:
The issue we'd run into here is that on hba a user is only connected to the jurisdiction that they created their account with but they can still log in to all of the sites. That whole situation is captured in https://github.com/bloom-housing/bloom/issues/4119 but I figured we'd want to handle that at once with some investigation around how common it is.

Reply:
Ohhhh interesting. Even from a UX standpoint, I think as a public user, I would expect that I would have to create different accounts for each public portal, but we can't really do that as is 🤔 Could we maybe check if it is equal to any of the appUrls on our stored jurisdictions?

Reply:
Ooo I really like that idea! My only hesitancy is that that data isn't available so it would require hitting the db again. Do you have any worries on the tradeoff between security and performance here @YazeedLoonat? If not, I'll make that update!

Reply:
the jurisdiction table is pretty small so look ups against it should be quick I don't think it'd be a problem to do that!

Reply:
Just updated, thanks for the feedback y'all 🌿